author.php

```diff
@@ -8,13 +8,36 @@
  * @subpackage  Timber
  * @since    Timber 0.1
  */
-global $wp_query;
+wp_register_script( 'author', get_template_directory_uri() . '/assets/js/author.js', [ 'jquery' ], '0.0.1', true );
+wp_enqueue_script( 'author' );
 
 $context = Timber::get_context();
 $context['posts'] = Timber::get_posts( false, 'P4_Post' );
+
+$post_args = [
+    'posts_per_page' => 10,
```

Comment:
Please add comma after last array element, also align the array double arrow.

Reply:
Done!